server_src/routes/index.js

```diff
@@ -42,8 +41,8 @@ console.log(`Version: ${version} (server started)`);
 
 // Welcome Page
 async function welcome() {
-  if (db === "") {
-    // No db setup, show db warning before login.
+  if (process.env.MONGO === "") {
```

Comment:
I dont really get why this welcome also checks mongo. Is it a leftover if-statement from the past?